app/main/views/audits.py

```diff
@@ -87,20 +87,25 @@ def list_audits():
             AuditEvent.user == user
         )
 
-    data_supplier_id = request.args.get('data-supplier-id')
+    # note in the following that even though the supplier and draft ids *are* integers, we're doing the searches as
+    # strings because of the postgres static type system's awkwardness with json types. we first let args.get normalize
+    # them into actual integers though
+
+    data_supplier_id = request.args.get('data-supplier-id', type=int)
```

Comment:
We have an existing test for the `data-supplier-id` param (https://github.com/alphagov/digitalmarketplace-api/blob/master/tests/main/views/test_audits.py#L904) - does it need updating or does it already cover the expected change?

Reply:
It would only make a difference for cases where it e.g. zero-prefixed the number for some reason.